activesupport/lib/active_support/core_ext/range/include_range.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
```

Comment:
The existence of this file is documented, and users may depend on it:

http://guides.rubyonrails.org/v5.2.0/active_support_core_extensions.html#include-questionmark

If we want to remove it, it needs a deprecation cycle first. Here's an example of how that might look:

https://github.com/rails/rails/blob/64c88fb5d2caf3c34742a07394ac68b8377c4936/activesupport/lib/active_support/core_ext/object/to_json.rb

Reply:
Thank you. Done.